server/app/routes/v1/stacks_api.rb

```diff
@@ -15,6 +15,22 @@ class StacksApi < Roda
       validate_access_token
       require_current_user
 
+      # @param [Stack] stack
+      # @param [Hash] data
+      def patch_stack(stack, data)
+        data[:stack_instance] = stack
+        outcome = Stacks::Patch.run(data)
+
+        if outcome.success?
+          audit_event(request, @grid, @stack, 'patch')
+          response.status = 200
+          {}
```

Comment:
First intuition would be to return HTTP 200 + `render('stacks/show')` to match the existing `PUT` behavior in other APIs.

Reply:
I think there are several variations in the API currently, both are fine by me, but generally I suppose it would be a good practice to return the object.

Reply:
✅ RESOLVED